docs/_manage/integration-performance/user-feedback.md

```diff
@@ -51,9 +53,7 @@ Any reported issue with no updates from you, Zapier Support, or end users for a
 
 ![Changelog Form with Issues](https://cdn.zappy.app/8afba17ecfb25faf6b87597e2cd54387.png)
 
-- You can also request issues to be closed manually by selecting one in the "Bugs & Feature Requests" tab of the developer platform, and leaving a comment to our Support team that the issue has been resolved. Provide info on what version number the fix is on, how to test the updates, and any other helpful callouts.
-
-- Once an issue is reviewed, it will either be closed or responded to with follow-up questions and feedback.
+- You can also request issues to be closed manually by updating their statuses to `Done`. Make sure updates are deployed and promoted to the lastest version of your integration before marking issues as `Done`, as it will be reviewed by a Zapier support member. See the [`Utilizing issue statuses` section](https://platform.zapier.com/manage/user-feedback#utilizing-issue-statuses) below for more details and best practices.
```

Comment:
lastest - typo :)

Reply:
I suggest editing to "Make sure updates are deployed, version promoted and users migrated if possible before marking issues..." as we often see devs forget to migrate.